psi4/src/psi4/detci/ciwave.h

```diff
@@ -41,12 +41,15 @@
 namespace psi {
 class Options;
 class JK;
-class DFERI;
 class IntegralTransform;
 class MOSpace;
 typedef std::shared_ptr<Matrix> SharedMatrix;
 class SOMCSCF;
 
+namespace df_helper {
+class DF_Helper;
+}
+
```

Comment:
@psi4/maintainers Out of curiosity how do we feel about this in a namespace? We usually leave it flat. Im not sure I really have a preference either way, but not being consistent isn't great.

Reply:
I would probably have gone with a flat namespace, but I don't feel strongly enough to request changing it now.  As long as there are no symbol clashes (which there aren't, obviously) I don't really have much of an opinion.